### DISCUSSION TITLE
Is this suppose to happen?

### DISCUSSION THREAD

**Discussion by jimmyz5150** [JurajNyiri/HomeAssistant-Tapo-Control#728] [General]
Hey I have a quick question
After HA update, my TPLink integration is now picking up my Tapo C310 (I use Tapo: Cameras Control) and HA sees it as a new device. TPLink I use as only switches and lights. TP doc says doesn't support C310. 
Any ideas what is going on?

HA states this below
3F62 C310 (192.x.x.x) 
TP-Link Smart Home

**Comment by JurajNyiri** [comment]
They are slowly adding support for cameras to the TPLink integration inside HA, the current implementation is very limited, until it matches what you need there is no reason to switch. You can use only 1 integration at the same time, so you have to pick either this, or the integrated HA one.

**Reply by jimmyz5150** [reply]
Ok good to know thanks. Was odd to see the TPLink show up. I like yours so will keep. 
Have a good night
